### PR TITLE
compatibility for python 3.8 on windows support

### PR DESCRIPTION
**Pull request by daniel-butler**
This sets the asyncio event loop policy per the tornado issue discussion.

### CODE REVIEW

**Comment by daniel-butler** [comment]
Related to issue #138

**Comment by daniel-butler** [comment]
Only python 2.7 fails

**Comment by jiffyclub** [comment]
Thanks for the PR! @embray are you able to test this out on Windows?

**Comment by embray** [comment]
I don't have Python 3.8 on Windows yet, but it seems legit to me.

File: snakeviz/cli.py [hunk 1]
```diff
@@ -108,9 +109,15 @@ def main(argv=None):
         parser.error('invalid port number %d: use a port between 0 and 65535'
                      % port)
 
+    # Before starting tornado set the eventloop policy for windows and python 3.8 compatibility
+    # https://github.com/tornadoweb/tornado/issues/2608
+    if sys.platform == 'win32':
+        asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```
Comment (eric-wieser) [inline snakeviz/cli.py]: Perhaps worth copying https://github.com/python/pyperformance/pull/65/files here?

Comment (jiffyclub) [reply snakeviz/cli.py:115]: Perfect, thanks for the tip!

**Comment by jiffyclub** [comment]
Thanks for this, I've incorporated it into master with ed88a7ef68d3826!

**Pull request closed without merging**